mw-webapp/src/styles/_variables.scss

```diff
@@ -5,6 +5,9 @@ $button-active: rgb(246, 246, 150);
 $button-radius: 12px;
 $button-margin: 20px;
 
+// Redirect Page
+$redirect-page-margin-top: 50px;
```

Comment:
Why?)

Reply:
forced...
![image](https://github.com/tritonJS826/masters-way/assets/95077794/f5a6c859-6520-4645-89fe-9f1916df3d07)

Reply:
Hahaha) So why didn't you discuss with them? Doing things only because someone tell you - bad style) Better way to understand why shoud we do something, what is our goals and will it help to achieve it) Dont hesitate to discuss even with me. Maybe they right, but we dont know it, because we don't know why did they ask you)

Reply:
Because I don’t want to argue or prove anything, since I’m practically not on the project

Reply:
We can finish your PR if you havn't time on this project. It's okey. We don't force you )